content: add session formats (#15)
ipfs/camp.ipfs.io: src/pages/formats.mdx

import { Page, Section } from '@layouts'

export default Page

export const title = 'Session Formats'

export const meta = {
  authors: ['foo', 'bar'],
  layout: Page,
  className: 'mw7',
}

# What will I learn at IPFS Camp?

### Session Types

- 👩🏽‍🏫 [Core & Elective Courses](#core)
- 📃 [Poster Sessions](#poster)
- 🐋 [Deep Dives](#deepdive)
- ⚡️ [Lighting Talks](#lighting)
- 🧩 [Unconf](#unconf)
- 🧬 [Sci-Fi Fair](#scifi)

<Section id="core">

## 👩🏽‍🏫 Core & Elective Courses

> The Courses of the InterPlanetary Space Training Program (aka IPFS Camp) are one of the main sessions for the event. These are designed to train new IPNauts and the more seasoned IPBuilders in all the arts of the DWeb and P2P.

Reviewer: ```suggestion
The Core and Elective Courses are the main sessions for the InterPlanetary Space Training Program (aka IPFS Camp). These are designed to train new IPNauts and more seasoned IPBuilders in the arts of the DWeb and P2P.
```

> The Courses are intensive workshops of 75 minutes each. The Core Courses will be a series of sessions to level up their understanding so that we all share a great baseline of understanding, while the Elective Courses will be on even more bleeding edge topics of the DWeb, where each participant decides which ones to attend to advance their knowledge of understanding.

Reviewer: ```suggestion
The Courses are intensive workshops of 75 minutes each. The Core Courses will be a series of sessions to level-up each participant's understanding so that we all share a great baseline to build upon. The Elective Courses will cover even more bleeding-edge topics of the DWeb, where each participant decides which ones to attend to advance their knowledge and deepen their understanding.
```

</Section>

<Section id="poster">

## 📃 Poster Sessions

> Poster Sessions are one of the all times IPFS Events favourites. These sessions are designed to propagate knowledge by having small teams of people explore together one of the many pieces of the IPFS stack with the goal to understand it well, so well that they can present it to the whole group aided by a Poster.

Reviewer: ```suggestion
Poster Sessions are one of the all times favorite IPFS Events. These sessions are designed to propagate knowledge by having small teams of people explore one of the many pieces of the IPFS stack together, with the goal to understand it so well that they can present it to the whole group aided by a Poster.
```

This session is inspired by the famous [Feynman Technique](https://collegeinfogeek.com/feynman-technique/), developed by Richard Feynman himself. The technique is simple and powerful, it suggests that for one to understand something well, it needs to be able to explain it.

</Section>

<Section id="lighting">

## ⚡️ Lighting Talks

> Lighting Talks are one of the most entertaining and inspiring sections of the IPFS Events. These serve the purpose to enable everyone that is attending the conference to share their own update on their projects with the whole community. Lighting Talks are designed for high throughput, high velocity and large coverage.

Reviewer: ```suggestion
Lighting Talks are one of the most entertaining and inspiring sections of IPFS Events. These serve to enable everyone attending the conference to share an update on their own projects with the whole community. Lighting Talks are designed for high throughput, high velocity, and large coverage.
```

Instructions

</Section>

<Section id="deepdive">

# 🐋 Deep Dives

> The Deep Dives format is designed to give all participants a taste of what it feels like to work on the big and chunky Open Problems by contributing directly to them in an intense design session, evaluating all the possible solutions, their shortcomings and documenting the results.

Reviewer: ```suggestion
The Deep Dives format is designed to give all participants a taste of what it feels like to work on the big and meaty Open Problems by contributing directly to them in an intense design session - evaluating all the possible solutions, their advantages and shortcomings, and documenting the results.
```

> This type of discussions are core to the genesis of IPFS. We’ve done multiple over the years which yield to many valuable results such as IPLD, PubSub, GraphSync and other advancements to the original IPFS protocol.

Reviewer: ```suggestion
This type of discussion is core to the genesis of IPFS. We’ve done multiple over the years which have yielded many valuable results such as IPLD, PubSub, GraphSync and other advancements to the original IPFS protocol.
```

> These design sessions are not about explaining how things work, they were focused on finding the solution with the knowledge and resources we had available.

Reviewer: ```suggestion
These design sessions are not about explaining how things work, they focus on finding the solution with the knowledge and resources we have available.
```

</Section>

<Section id="unconf">

## 🧩 Unconf

We will host an Unconf segment during IPFS Camp to create space for all the conversations, talks, discussions and idea sharing that didn’t get the opportunity to be discussed during the other sessions.
Instructions

</Section>

<Section id="scifi">

## 🧬 Sci-Fi Fair

> Do you fancy Science Fairs? Always felt that there was something missing? Something even more futuristic? Despair no more and welcome to the 🧬 IPFS Camp Sci-Fi Fair 🛸. The Sci-Fi fair was created to give a free roaming space for all our attendees to meet all sorts of projects happening from the other attendees of the event.

Reviewer: ```suggestion
Do you fancy Science Fairs? Always felt that there was something missing? Something even more futuristic? Despair no more and welcome to the 🧬 IPFS Camp Sci-Fi Fair 🛸. The Sci-Fi fair was created to give a free roaming space for all our attendees to meet the awesome projects from other event attendees.
```
